cmd/juju/user/change_password.go

```diff
@@ -117,10 +118,13 @@ func (c *changePasswordCommand) Run(ctx *cmd.Context) error {
 			return errors.Trace(err)
 		}
 		c.api = api
-		defer c.api.Close()
 	}
+	defer c.api.Close()
```

Comment:
it was in the right place, in the block that created the client (NewUserManagerAPIClient). please move it back